integrations/apollo/src/index.ts

```diff
@@ -124,6 +127,20 @@ export const createApolloClient = ({
       )
     : httpLink
 
+  const links = [];
+
+  if(onError) {
```

Comment:
This looks incorrectly formatted. Please run Prettier locally. I was under the impression that we had a git hook for this, but apparently, we don't 🤔

Reply:
Done, nice if you can handle the formatting 💯